.gitignore

```diff
@@ -377,6 +377,9 @@ client_assertions.txt
 *.manifest
 *.otbm
 *.rar
+*-house.xml
```

Comment:
Why removing xmls from versioning ?

Reply:
Do this also ignore changes to already added files? (thought it didn't)

Reply:
I think, when we add some file in gitignore. the files can be removed from repository and future files will not be added in repository either.

Reply:
I dont think this.
For example. We ignore *.otbm files. (see row 378)
But changes to the otservbr-custom.otbm-file is still under version control. Since this file has been added for version control.

My idea here was to be able to update existing files but ignore additional files.

Reply:
we removed the .otbm because github doesn't work well with binary, every time you change a binary and upload it to github it makes a copy of it so the size of the repository can increase a lot.

Reply:
I understand this. But this doesnt change how the ignore works. 
I added rows to the gitignore since I assumed:

1) We wont add many or any new maps with npc/house/monster-xml-files
2) If we add any we can always add them for version control
3) People testing want to have several maps in custom folder without accidentaly pushing npc/house/monster files

I could always specify that we only ignore these files if they are in the custom map folder.

Reply:
Particulary, in my repository, i made all otbm with versioning, just global map, is in gitignore, because it's a large file.
ex:

```
## CUSTOM
# Files
#config.lua
client_assertions.txt
# Extensions
*.ini
*.otb
*.exe
*.manifest
#*.otbm
*.rar
otservbr-byelson.otbm
``` 

otservbr-byelson.otbm it's my global map.

Reply:
Sure, this works but then everybody needs to have their own modified gitignore in their server branch. 
I am open for feedback, but dont see a problem with having this in the gitignore. 
Guess Admins decide.